uploader/src/main/resources/cfn-template.yaml

```diff
@@ -254,6 +254,14 @@ Resources:
                         Bitrate: 160000
                         CodingMode: CODING_MODE_2_0
                         SampleRate: 44100
+              - ContainerSettings:
+                  Container: RAW
+                VideoDescription:
+                  CodecSettings:
+                    Codec: FRAME_CAPTURE
+                    FrameCaptureSettings:
+                      MaxCaptures: 1
+                      Quality: 95
```

Comment:
Serving this high quality directly to users might cause issues. If the images end up in `guim` that's okay, as the Fastly resizer can handle them. Do we know where these images end up?

Of course, I imagine in most cases we'll be serving these images internally (as previews for self-hosted videos in Media Atom Maker and Fronts Tool), so it's not an issue in these cases, but if we end up using it as a 'first frame' to load for poor internet conditions we would probably want the Fastly resizer to be doing its work.